app/services/date_time_formatter.rb

```diff
@@ -1,20 +1,20 @@
 # frozen_string_literal: true
 
-class DateFormatter
+class DateTimeFormatter
   def self.month_day_time(datetime)
-    datetime.strftime("%A, %B %e at %l %p %Z")
+    datetime.strftime("%A, %B %e at %l:%M %p %Z")
   end
 
   def self.month_day(datetime)
     datetime.strftime("%A, %B %e")
   end
 
   def self.time_with_zone(datetime)
-    datetime.strftime("%l:%m %p %Z")
+    datetime.strftime("%l:%M %p %Z")
   end
 
   def self.time(datetime)
-    datetime.strftime("%l:%m %p")
+    datetime.strftime("%l:%M %p")
```

Comment:
Maybe we could use some attributes with friendly names to make the formats more readable and reduce the `datetime.strftime` repetition:

```ruby
private

@format_month_day = "%A, %B %e"
@format_time = "%l:%M %p"
@format_month_day_at_time = "#{@format_month_day} at #{@format_time_with_timezone}"
@format_time_with_timezone = "#{@format_time} %Z"

self.format_time(datetime, format)
  datetime.strftime(format)
end
```

Reply:
I'm not sure if I understand what you're going for here. Could you show me an example of how you'd use this?